style.css

```diff
@@ -0,0 +1,178 @@
+@import 'https://fonts.googleapis.com/css?family=Crete Round|Inter';
+
+body {
+  background-color: #111;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  color: #f8f8f8;
+  overflow-x: hidden;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  padding: 7px 100px;
+  background-color: #444;
+}
+
+header .brand a {
+  font-size: 25px;
+  text-decoration: none;
+  color: #f8f8f8;
+}
+
+header .nav-menu ul {
+  display: flex;
+}
+
+header .nav-menu ul li {
+  list-style: none;
+  padding-left: 30px;
+}
+
+header .nav-menu ul li a {
+  text-decoration: none;
+  color: #f8f8f8;
+  font-size: 18px;
+  font-weight: 600;
+  transition: 0.5s;
+}
+
+header .nav-menu .nav-button {
+  display: none;
+}
+
+.headline .social-network ul li a {
+  text-decoration: none;
+  color: #f8f8f8;
+}
+
+header .nav-menu ul li a:hover {
+  padding-bottom: 10px;
+  border-bottom: solid 3px #3189ec;
+}
+
+/* ====== Headline ====== */
+
+.headline {
+  display: flex;
+  align-items: center;
+  margin-top: 30px;
+  height: calc(100vh - 70px);
+}
+
+.headline .social-network ul {
+  padding-right: 40px;
+}
+
+.headline .social-network ul li {
+  list-style: none;
+  width: 25px;
+  height: 25px;
+  overflow: hidden;
+  margin-top: 10px;
+  transition: 0.5s;
+}
+
+.headline .social-network ul li:hover {
+  transform: scale(1.2);
+}
+
+.headline .image img {
+  width: auto;
+  height: 100%;
+}
+
+.headline .social-network ul li a img {
+  width: 100%;
+  height: 100%;
+}
+
+.headline .resume {
+  flex-basis: 40%;
+  padding-left: 30px;
+}
+
+.headline .resume h1 {
+  font-size: 3rem;
+  padding: 0 !important;
+  margin: 0 !important;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  color: #ff6b00;
+}
+
+.headline .resume h4 {
+  font-size: 1.5rem;
+  font-family: Inter, sans-serif;
+  font-weight: 500;
+}
+
+.headline .resume p {
+  font-size: 20px;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+}
+
+.headline .image {
+  position: absolute;
+  top: 120px;
+  right: 10px;
+  flex-grow: 1.5;
+  display: flex;
+  justify-content: flex-end;
+  height: 550px;
+}
+
+@media screen and (max-width: 375px) {
```

Comment:
- [ ] Kindly resize your media query breakpoint, currently it's too small ,check out [this article](https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/) to learn common breakpoint